src/Server.ts

```diff
@@ -0,0 +1,25 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import express from 'express';
+import path from 'path';
+
+import { Logger } from './logger/Logger';
+import { userRoute } from './routes/UserRoutes';
+import { authenticationRoute } from './routes/AuthenticationRoutes';
+
+const server = express();
+
+server.listen(5000, () => {
+    console.log('🚀 Server is running');
+});
+
+server.use(Logger.consoleLog);
+server.use(express.static(path.resolve(__dirname, './api/view/html/public')));
+
+server.use('/api', userRoute);
+server.use('/auth', authenticationRoute);
+
+server.all('*', (req, res) => {
```

Comment:
Please add a comment here explaining what this does.

Reply:
I don't think comments are needed there because that's like the usual and expected format in express. It does the same thing other routes do, except instead of the static route it applies to "everything else". 

Once again, I don't mind adding comments but then there should be guidance regarding the parts that should be explained by the comments because this line is understood by anyone who uses express. If we are going to comment this we would also need to comment on every other part of the express routing statements.